src/main/java/org/jenkinsci/test/acceptance/plugins/credentialsbinding/CredentialsBinding.java

```diff
@@ -0,0 +1,32 @@
+package org.jenkinsci.test.acceptance.plugins.credentialsbinding;
+
+import org.jenkinsci.test.acceptance.po.ContainerPageObject;
+import org.jenkinsci.test.acceptance.po.Control;
+import org.jenkinsci.test.acceptance.po.PageArea;
+import org.jenkinsci.test.acceptance.po.PageAreaImpl;
+
+/**
+ * Represents a credential binding in the job configuration page.
+ */
+public class CredentialsBinding extends PageAreaImpl {
+
+    public Control credentialId = control("credentialId");
+    public Control variable = control("variable");
+    
+    public CredentialsBinding(PageArea area, String path) {
+        super(area, path);
+    }
+    
+    public CredentialsBinding(ContainerPageObject po, String path) {
+        super(po, path);
+    }
+    
+    /**
+     * Checks whether there are credentials in the credentials drop down
+     * 
+     * @return true if there are no credentials, false otherwise
+     */
+    public boolean noCredentials() {
+        return getElement(by.tagName("options")) == null;
```

Comment:
oops, you are right, I meant `option`, that's a typo. And you are right, it is looking for `<option>` tags in the whole page. I did not intend to do that, I wanted to look for it in the PageArea context. Will fix. Thanks